api/validation/pokemons.validation.ts

```diff
@@ -15,10 +16,52 @@ function validateGetRandomPokemon(
   });
 
   const { error } = schema.validate(req.query);
-
   if (error) {
     return res.status(400).json({ message: error.details[0].message });
   }
 
   next();
 }
+
+function validateGetPokemons(req: Request, res: Response, next: NextFunction) {
+  const querySchema = Joi.object({
+    sort: Joi.string()
+      .valid(
+        "Alphabetical A-Z",
+        "Alphabetical Z-A",
+        "Power (High to low)",
+        "Power (Low to high)",
+        "HP (High to low)",
+        "HP (Low to high)"
```

Comment:
dont pass query params with spaces.
why dont you pass two parameters? 
sort_order
sort_by